Tests/SubscriberTests/WorkerQueue/SubscriberWorkerFactoryTests.swift

```diff
@@ -0,0 +1,34 @@
+import XCTest
+import AblyAssetTrackingInternal
+import AblyAssetTrackingInternalTesting
+@testable import AblyAssetTrackingSubscriber
+import AblyAssetTrackingSubscriberTesting
+
+class SubscriberWorkerFactoryTests: XCTestCase
+{
+    private let logHandler = InternalLogHandlerMock.configured
+    private var legacyWorkerCalled = false
```

Comment:
Can't this be a local variable in the test function?

Reply:
8a90707